core-plugins/src/test/java/co/cask/hydrator/plugin/transform/PythonEvaluatorTest.java

```diff
@@ -185,7 +160,7 @@ public void testSchemaValidation() throws Exception {
     // check if schema is null, input schema is used.
     config = new PythonEvaluator.Config(
       "def transform(x, emitter, context):\n" +
-        "  y = {}\n" +
+        " y = {}\n" +
```

Comment:
Yea, I just did a git revert of the referenced PR/commit.
We can revert this revert once the linked JIRA for memory leak is resolved.